platform/windows/vulkan_context_win.cpp

```diff
@@ -28,6 +28,9 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+// NOTE: disabling entire file contents instead of removing from sources in build file to remain consistent with other files in this directory
```

Comment:
This is done throughout the Godot codebase, I don't think we want to replicate this comment in a few dozen files.

Reply:
I read in the style guide to put a comment into the code explaining why you are changing something.  Then I was told the opposite in the developer chat channels.   Now I know.

Reply:
removed